setup.py

```diff
@@ -2,16 +2,16 @@
 
 setup(
     name="imhotep",
-    version="3.0.0",
+    version="3.0.1",
     packages=find_packages(),
-    url="https://github.com/justinabrahms/imhotep",
+    url="https://github.com/eBay/imhotep",
     license="MIT",
     author="Justin Abrahms",
     author_email="justin@abrah.ms",
     description="A tool to pipe linters into code review",
     python_requires=">=3.9",
     install_requires=[
-        "requests~=2.20.0",
+        "requests",
```

Comment:
What's the thinking here?

Reply:
We’re using a very basic functionality of `requests`. Requiring too strict a version makes new plugins difficult to develop. (For example, when I was developing my `imhotep_pmd`, I wanted to use a new version of this package, and there was a conflict.)